app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/contact-information.gjs

```diff
@@ -0,0 +1,84 @@
+import Component from "@glimmer/component";
+import { Input } from "@ember/component";
+import { action } from "@ember/object";
+import DButton from "discourse/components/d-button";
+import i18n from "discourse-common/helpers/i18n";
+import GroupChooser from "select-kit/components/group-chooser";
+import UserChooser from "select-kit/components/user-chooser";
+
+export default class AdminConfigAreasAboutContactInformation extends Component {
+  @action
+  save() {
+    this.args.saveCallback();
+    // eslint-disable-next-line no-console
+    console.log("contact information");
+  }
+
+  <template>
+    <div class="control-group">
+      <label>
+        <span>{{i18n "admin.config_areas.about.community_owner"}}</span>
+        <span class="admin-config-area-card__label-optional">{{i18n
+            "admin.config_areas.about.optional"
+          }}</span>
+      </label>
+      <p class="admin-config-area-card__additional-help">
+        {{i18n "admin.config_areas.about.community_owner_help"}}
+      </p>
+      <Input />
```

Comment:
for all new code I recommend using native `<input>` over ember's old Input component

```suggestion
      <input type="text />
```